Channels/UserInterface/04c_ChannelsPIM.md

```diff
@@ -0,0 +1,307 @@
+
+      - ![Private view](../../Assets/Icons/Unpublish.png "[Unpublish]") private view    
+        Click this entry to unpublish the view. This menu entry is only displayed if a view has been selected and published.
+
+      - ![Save](../../Assets/Icons/Save.png "[Save]") save    
+        Click this entry to save the current view settings in the selected view. This menu entry is only displayed if a view has been selected.
+
+        > [Info] When the settings of a view have been changed, an asterisk is displayed next to the view name. The asterisk is hidden as soon as the changes have been saved.
+
+      - ![Delete](../../Assets/Icons/Trash01.png "[Delete]") delete    
+        Click this entry to delete the selected view. A confirmation window is displayed to confirm the deletion. This menu entry is only displayed if a view has been selected.
+
+
+- Selected connection    
+  The selected connection is displayed in the drop-down list. The drop-down list is locked.
+
+- ![Columns](../../Assets/Icons/Columns.png "[Columns]") Columns (x)   
+  Click this button to display the columns bar and customize the displayed columns and the order of columns in the list. The *x* indicates the number of columns that are currently displayed in the list.
+
+- ![Filter](../../Assets/Icons/Filter.png "[Filter]") Filter (x)   
+Click this button to display the filter bar and customize the active filters. The *x* indicates the number of filters that are currently active.
+
+- [x]     
+  Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all offers in the list are selected.
+
+- [CREATE PIM PRODUCTS/MAP OFFERS TO EXISTING PIM PRODUCTS]            
+  Click this button to import the selected offers in the list. The *Summary* window is displayed, see [Summary](#summary-1).
+
+
+The list displays all offers to be imported. Depending on the settings, the displayed columns may vary. All fields are read-only. 
+
+- *SKU*   
+  Stock Keeping Unit. Identification number of the product to the offer. 
+ 
+- *Variant status*  
+  Variant type of the offer. The following options are available:  
+  - **Master offer**   
+    This offer refers to a master product.
+  - **Variant offer**   
+    This offer refers to a variant.
+  - **Standalone Offer**  
+    This offer refers to a single product.
+
+- *Status*  
+  Status of the offer. The following options are available:  
+  - **Active**   
+    The offer is active. It is displayed in the marketplace, where it can be sold.   
+  - **Inactive**   
+    The offer is inactive. It is not longer displayed in the marketplace and thus cannot be sold anymore.   
+  - **Not available**   
+    The offer is deleted from the marketplace.   
+    The *Change offer status* confirmation window is displayed. If the variant status of the selected offer equals **Master offer**, an additional toggle is displayed within the confirmation window.   
```

Comment:
"within the confirmation window"? Bedeutet das ein "Change offer status" Fenster mit üblicher "Are you sure...?" Frage + toggle zusäztlich, wenn offer ist ein Master offer? 
Im Screenshot sieht es so wie zwei unterschiedliche Fenster aus? Sind sie zwei Fenster, die nacheinander/nebeneinander vorkommen (können, je nach offer variant)?